pkt-line.c

```diff
@@ -426,7 +426,7 @@ char *packet_read_line_buf(char **src, size_t *src_len, int *dst_len)
 	return packet_read_line_generic(-1, src, src_len, dst_len);
 }
 
-ssize_t read_packetized_to_strbuf(int fd_in, struct strbuf *sb_out)
+ssize_t read_packetized_to_strbuf(int fd_in, struct strbuf *sb_out, int options)
```

Comment:
nit: `options` -> `flags`? I'm not set on this, so feel free to ignore.

Reply:
Oh, you're absolutely right!

Reply:
... except that we actually hand that value to `packet_read()`, which _already_ takes an `options` parameter... So I'll leave this consistent with the rest of `pkt-line.c`.